### PR TITLE
Fix two splicing/vls omissions

### PR DESCRIPTION
**Pull request by ksedgwic**
I think we lost these two deltas while rebasing, apologies ...

### CODE REVIEW

File: channeld/channeld.c [hunk 1]
```diff
@@ -5378,7 +5378,9 @@ static void handle_funding_depth(struct peer *peer, const u8 *msg)
 			peer_write(peer->pps, take(msg));
 
 			peer->channel_ready[LOCAL] = true;
-		} else if (splicing && !peer->splice_state->locked_ready[LOCAL]) {
+			check_mutual_channel_ready(peer);
+		}
+		else if(splicing && !peer->splice_state->locked_ready[LOCAL]) {
```
Comment (vincenzopalazzo) [inline channeld/channeld.c:5383]: we can move the `else if` onf the same line of `}`?

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/6834/commits/b45e2f26be8f4b0d3a8d27a70be85403bc2e9673#diff-b1299b24697159a06e7b426c5086c36df8e48db15739a0e301d53a9a21149722R5382-R5383

Good catch, it is my fault sorry about that

File: channeld/channeld.c (hunk 1 above)
Comment (ksedgwic) [reply channeld/channeld.c]: done

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/6834/commits/6193b5d8f586f8f546ec99c7770876377e7df0de

**Review by nepet — APPROVED** [review]
ACK 6193b5d8f586f8f546ec99c7770876377e7df0de

**Pull request merged**